packages/conjure-client/src/errors/error.ts

```diff
@@ -50,8 +50,12 @@ export class ConjureError<E> {
 }
 
 export function isConjureError(error: any): error is ConjureError<never> {
+    const conjureErrorTypes = Object.keys(ConjureErrorType).map(
```

Comment:
theres only handful of fields, maybe just check them explicitly?

Reply:
I think that risks that when someone adds a new enum value, they forget to add the new value here.

Reply:
your implementation has the same risk since it doesn't even verify that the types of the field is correct

Reply:
Why? What type do I not verify? I check if the provided `type` field is a valid enum value. If you extend the enum, that value will also be accepted. Other values are not accepted.